pulsar-client-go/pulsar/producer_test.go

```diff
@@ -416,3 +416,71 @@ func TestProducer_SendAndGetMsgID(t *testing.T) {
 		assert.NotNil(t, IsNil(msgID))
 	}
 }
+
+func TestProducer_DelayMessage(t *testing.T) {
+	client, err := NewClient(ClientOptions{
+		URL: "pulsar://localhost:6650",
+	})
+	assert.Nil(t, err)
+	defer client.Close()
+
+	topicName := "test-send-with-message-id"
```

Comment:
Can we rename the topic name?

Reply:
This topic name is the same as the topic name in another test case, in order to avoid test conflicts, please replace it

Reply:
```
func newTopicName() string {
	return fmt.Sprintf("my-topic-%v", time.Now().Nanosecond())
}
```

Reply:
ofcourse